projects/network_table_listener/NetworkTableListener.py

```diff
@@ -0,0 +1,239 @@
+from nt_connection.Connection import ConnectionError
```

Comment:
Can you also add a brief summary about what this file does at the top. 
Example from land_satellite_listener:

```
"""land_satellite_listener.py: Facillitates communication between Land Server and BBB.""" 
   
 __author__ = "Brielle Law (briellelaw), John Ahn (jahn18)" 
 __copyright__ = "Copyright 2020 UBC Sailbot"  
```

Reply:
done